src/components/axes/timeAxis.ts

```diff
@@ -1,6 +1,11 @@
 ///<reference path="../../reference.ts" />
 
 module Plottable {
+
+export enum TimeInterval {
+  second, minute, hour, day, week, month, year
```

Comment:
`string`y enums also have the advantage of using the raw string values for those insistent on using raw strings (usually Javascript consumers)